### PR TITLE
Add new section "Deployment and Administration Notes"

### PR DESCRIPTION
**Pull request by cjschroder**
re https://github.com/SUSE/doc-cap/issues/292

### CODE REVIEW

**Comment by cjschroder** [comment]
I'm thinking this can expand over time, and we can xref instead of printing the same information multiple times

File: xml/cap_depl_install_production.xml [hunk 1]
```diff
@@ -41,7 +41,43 @@
...
+<sect1 xml:id="sec.cap.deploy-notes">
+  <title>Deployment and Administration Notes</title>
+  <para>
+      This section lists considerations for deploying and operating
+      &productname;.
+  </para>
+  
+  <variablelist>
+   <varlistentry>
+       <term>Length of release names limits</term>
```
Comment (btat) [inline xml/cap_depl_install_production.xml]: Would "Length of release names" or "Release name length" work here? Labeling it "limits" makes the product sounds so...limiting heh

Comment (cjschroder) [reply xml/cap_depl_install_production.xml]: ee such clumsy wording :)

**Comment by btat** [comment]
:+1:

**Pull request merged**